src/main/java/com/openlattice/chronicle/api/ChronicleApi.java

```diff
@@ -146,6 +147,22 @@ ChronicleQuestionnaire getChronicleQuestionnaire(
             @Path( ENTITY_KEY_ID ) UUID questionnaireEKID
     );
 
+    /**
+     * Send Message to participant.
+     *
+     * @param organizationId    - Id of the organization to which study belongs
+     * @param studyId           - the studyId
```

Comment:
```suggestion
     * @param studyId           - the studyId
     * @param participantId - participant id
```

Reply:
please use our standard formatter(https://github.com/openlattice/openlattice/blob/main/eclipse-code-format-settings.xml)  to configure your ide

Reply:
> 

I suspect running code reformatting won't agree with this suggestion

Reply:
GitHub doesn't run the auto-formatter. This is just an inline suggestion.

Reply:
oh just realized that the @param participantId was missing. Should still run the formatter though.